ui/src/views/Documentation/Reference/index.jsx

```diff
@@ -62,6 +66,27 @@ export default class Reference extends Component {
             ))}
           </Fragment>
         )}
+        {logTypes && Boolean(logTypes.length) && (
+          <Fragment>
+            <Typography component="h2" variant="h5">
+              Message Types
+            </Typography>
```

Comment:
Switch that to `HeaderWithAnchor`. You'll need to pull recent changes from upstream :)